pkg/combustion/registry.go

```diff
@@ -207,8 +209,13 @@ func registryArtefactsPath(ctx *image.Context) string {
 }
 
 func populateRegistry(ctx *image.Context, images []string) error {
+	p, err := podman.New(ctx.BuildDir)
+	if err != nil {
+		zap.S().Warnf("Setting up Podman instance: %v", err)
```

Comment:
We cannot proceed execution because `p` is nil after this. Shouldn't we just error out?